dlgr/griduniverse/static/scripts/dist/.gitattributes

```diff
@@ -0,0 +1,4 @@
+*.js -diff -merge
```

Comment:
I must have just had the file in the wrong place(?) I had it at the repository root.

Reply:
It's OK to have in the repository root, but then the glob paths are more difficult I think,